phase5/Solution1/ConsoleApp1/Runnable.cs

```diff
@@ -0,0 +1,58 @@
+using SampleLibrary;
+
+namespace ConsoleApp1;
+
+public class Runnable
```

Comment:
`Runnable` name is not much that meaningful for a class! (personally remember that we use such a name for interfaces in Java)